core/src/main/resources/reference.conf

```diff
@@ -138,8 +138,9 @@ akka.persistence.cassandra {
     # enabled and the keyspace doesn't exist already.
     data-center-replication-factors = []
 
-    # Maximum number of messages that will be batched when using `persistAsync`.
-    # Also used as the max batch size for deletes.
+    # Maximum total size in bytes of serialized messages that will be grouped in one batch when writing multiple
```

Comment:
where is it used as bytes? Note that `payload.size` is not bytes, that is still a `Seq[Serialized]`

Reply:
🤦 my bad from not reading sources well enough